app/src/main/java/com/sildev/thecocktail/data/model/Category.kt

```diff
@@ -0,0 +1,8 @@
+package com.sildev.thecocktail.data.model
+
+import com.google.gson.annotations.SerializedName
+
+data class Category(
+    @SerializedName("strCategory")
+    val name: String
```

Comment:
```suggestion
    val name: String?
```
nhớ check null khi. sử dụng gson converter